runtime/rasdump/javadump.cpp

```diff
@@ -1776,6 +1782,54 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 		"NULL\n"
 		"NULL           ------------------------------------------------------------------------\n"
 	);
+#if defined(J9ZTPF)
+	struct immap_privatemstats ztpfNativeStats;
+	int rc;
+
+	if (MAP_FAILED != mmap(NULL, 0, 0, MAP_SUPPORTED, 0, 0)) {
+		rc = mprivatestats(&ztpfNativeStats);
+
+		_OutputStream.writeCharacters("0SECTION         Native configuration information for z/TPF\n");
+		_OutputStream.writeCharacters("NULL             ==========================================\n");
```

Comment:
The second word should start in column 16 like other output lines:
```
|234567890123456
0SECTION       Native configuration information for z/TPF
NULL           ==========================================
```
Likewise in each call to `writeCharacters()` below.